modules/primer-dropdown/README.md

```diff
@@ -0,0 +1,312 @@
+# Primer CSS dropdown
+
+[![npm version](http://img.shields.io/npm/v/primer-dropdown.svg)](https://www.npmjs.org/package/primer-dropdown)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+This repository is a module of the full [primer-css][primer-css] repository.
+
+## Install
+
+This repository is distributed with [npm]. After [installing npm][install-npm], you can install `primer-dropdown` with this command.
+
+```
+$ npm install --save primer-dropdown
+```
+
+## Usage
+
+The source files included are written in [Sass][sass] (`scss`) You can simply point your sass `include-path` at your `node_modules` directory and import it like this.
+
+```scss
+@import "primer-dropdown/index.scss";
+```
+
+You can also import specific portions of the module by importing those partials from the `/lib/` folder. _Make sure you import any requirements along with the modules._
+
+## Build
+
+For a compiled **css** version of this module, a npm script is included that will output a css version to `build/build.css` The built css file is also included in the npm package.
+
+```
+$ npm run build
+```
+
+## Documentation
+
+<!-- %docs
+title: Dropdown
+status: Stable
+-->
+
+Dropdowns are lightweight, JavaScript-powered context menus for housing navigation and actions. They're great for instances where you don't need the full power (and code) of the select menu.
```

Comment:
> Should we say something here about the JavaScript not being included in this package, or suggest the use of `<details>` and `<details-menu>`?

For [`.Box-overlay`](https://styleguide.github.com/primer/components/box-overlay/) I just used `<details-dialog>` directly and basically imply that it is a mandatory dependency because it is an **essential** component for accessibility. 

While for `<details-menu>` isn't as necessary since menus are still _accessible_ without the added keyboard navigations, albeit can be better. This depends on how we want to word things for the different groups of style guide audiences. IMO for dotcom developers I'd suggest to always use `<details-menu>`, but I don't know for the general public.

> I'd love some input on whether the ARIA attributes (aria-expanded and aria-haspopup) are necessary in that case.

`aria-expanded` is handled by `<summary>` natively so we don't need it.

`aria-haspopup` is needed cause AT would then announce it as "menu popup button". `aria-haspopup="true"` defaults to `menu`. I prefer to be explicit and do `aria-haspopup="menu"` directly. 

When used with `<details-menu>` [this attribute is added via the custom element.](https://github.com/github/details-menu-element/blob/0df011210f7823efc1daab06dfd7fe3ddf5fcd8d/index.js#L23)

`aria-haspopup="dialog"` is another possible value, which we uses with `<details-dialog>`.